src/fpmath.h

```diff
@@ -109,14 +98,14 @@ union IEEEf2bits {
 union IEEEd2bits {
 	double	d;
 	struct {
-#if _BYTE_ORDER == _LITTLE_ENDIAN
-#if _IEEE_WORD_ORDER == _LITTLE_ENDIAN
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#if _IEEE_WORD_ORDER == __ORDER_LITTLE_ENDIAN__
```

Comment:
At least GCC provides __FLOAT_WORD_ORDER__ for this purpose. Would it make sense to also use this name directly instead of using _IEEE_WORD_ORDER?